airdrop/utils.gno

```diff
@@ -0,0 +1,175 @@
+package airdrop
+
+import (
+	"gno.land/p/demo/ufmt"
+)
+
+const (
+	MaxUint64 = 1<<64 - 1
+	uintSize  = 32 << (^uint(0) >> 63)
+)
+
+func lower(c byte) byte {
+	return c | ('x' - 'X')
+}
+
+// TODO: Remove parseUint after gno supports strconv.ParseUint
+func parseUint(s string, base int, bitSize int) (uint64, error) {
```

Comment:
Does it have to `ParseUint` ? 
Gno supports `Itoa`, so..

Reply:
In most cases, it is correct to use the `strconv.Atoi` function. However, `claimedBitmap` is currently stored as bytes, and there isn't yet a function in gno's `strconv` package that formats bytes. Therefore, I added `parseUint` function. I've modified the rest of the code to use `strconv` package, except for one specific part.

Reply:
https://github.com/gnoswap-labs/governance/pull/19/commits/0b44c70f44113b96b3849a6a8bca43614a1c4be3

Reply:
How about using `ufmt`, is it possible?

Reply:
Probably not, since this need to output in byte format. But since `parseUint` follows the specification of `strconv.ParseUint` function, so we could change it without trobule if pr [1464](https://github.com/gnolang/gno/pull/1464) merged in the future.